src/containers/NewPositionWrapper/NewPositionWrapper.tsx

```diff
@@ -465,6 +465,30 @@ export const NewPositionWrapper: React.FC<IProps> = ({
     return BigInt(0)
   }
 
+  const onRefresh = () => {
```

Comment:
what do you think about to add also refresh of balances from wallet and price from coingecko,